### PR TITLE
Testing 2

### PR DESCRIPTION
**Pull request by DavyKn**
edit testing added

### CODE REVIEW

File: src/add_remove.test.js [hunk 1]
```diff
@@ -12,6 +12,18 @@ describe('Add, Remove Element Test', () => {
...
   });
 
+  test('Add one new item to the list', () => {
+    document.body.innerHTML = '<div class="list-elements">'
+
+    + '</div>';
+    addElement('Cooking');
+    const list = document.querySelectorAll('.list-elements');
+    expect(list).toHaveLength(1);
+    // expect(list).toHave(`
```
Comment (Ismailco) [inline src/add_remove.test.js]: - [ ] Kindly consider removing all unesessery commented lines of code from your code, so it's clean and more readable

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @DavyKn, @ice949, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ All pure functions are tested
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/add_remove.test.js (hunk 1 above)
Comment (ice949) [reply src/add_remove.test.js]: @Ismailco Thanks we on it

Comment (Whoistolu) [reply src/add_remove.test.js]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**